BUILDING.md

```diff
@@ -94,20 +96,45 @@ Then, run the system preparation scripts and try to compile the project:
 cp components/hab/install.sh /tmp/
 sh support/linux/install_dev_0_ubuntu_latest.sh
 sh support/linux/install_dev_9_linux.sh
-. ~/.profile
+```
+
+When you run the install_dev_9_linux.sh script, you may see this error.
```

Comment:
I think we should fix the `install_dev_0_ubuntu_latest.sh` script ather than telling people to manually build and install libreadline7.

The cause of the problem is self-inflicted, since we are installing packages from Ubuntu 17.04 into 16.10. 
https://github.com/habitat-sh/habitat/blob/master/support/linux/install_dev_0_ubuntu_latest.sh#L32-L38

The script should check if the version of libreadline we want is available via apt, and if not then we canmanually grab and install the packages and deps we want.